plugins/chat/app/services/chat/create_thread.rb

```diff
@@ -30,50 +30,47 @@ class CreateThread
     policy :threading_enabled_for_channel
     model :original_message
     transaction do
-      step :find_or_create_thread
+      model :thread, :find_or_create_thread
       step :associate_thread_to_message
-      step :fetch_membership
```

Comment:
Moved these three steps outside the transaction, as they’re not related to the thread creation.
Also changed from `step` to `model` for fetching the membership (from its name, it was pretty obvious to use a `model` step :grin:)